pages/_app.tsx

```diff
@@ -1,7 +1,9 @@
 import '../styles/globals.scss'
 import type { AppProps } from 'next/app'
 
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Can any other thing be done to prevent this workaround?
Try to look for solutions.

Reply:
Ok bhaiya ! I will explore a standard workaround.